ecommerce_integrations/unicommerce/invoice.py

```diff
@@ -381,8 +381,9 @@ def create_sales_invoice(
 	si.naming_series = channel_config.sales_invoice_series or settings.sales_invoice_series
 	si.delivery_date = so.delivery_date
 	si.ignore_pricing_rule = 1
-	si.update_stock = update_stock
+	# si.update_stock = update_stock
```

Comment:
Why is this disabled? this is how other users use it. 

Don't just change code to achieve what you want, the integration is used by many users differently. You have to make everything configurable and default behaviour should continue to be there. 

Why you need delivery notes separately anyway?

Reply:
No prior delivery note had been created,
and the stock is directly updated on the sale invoice

Now it's configure with Delivery note settings